components/webgpu/lib.rs

```diff
@@ -330,6 +331,12 @@ impl WebGPU {
     }
 }
 
+type WGPUBufferMaps<'a> =
+    HashMap<id::BufferId, Rc<BufferMapInfo<'a, Option<WebGPUResponseResult>>>>;
+type WGPUPresentBufferMaps<'a> =
+    HashMap<id::BufferId, Rc<BufferMapInfo<'a, (Option<ErrorScopeId>, WebGPURequest)>>>;
```

Comment:
```suggestion
type WebGPUBufferMaps<'a> =
    HashMap<id::BufferId, Rc<BufferMapInfo<'a, Option<WebGPUResponseResult>>>>;
type WebGPUPresentBufferMaps<'a> =
    HashMap<id::BufferId, Rc<BufferMapInfo<'a, (Option<ErrorScopeId>, WebGPURequest)>>>;
```

Just for consistency with the rest of the code.